collections/_blogs/2021-01-25-announcing-meshery-v0.5.0.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+date:   2021-01-25 08:00:00 +0000
+image: /assets/images/posts/2020-08-07-announcing-meshery-v040/meshery-v040.png
```

Comment:
Should we use the special version of the image to meshery like `meshery 0.5.0 release`?